src/node.cc

```diff
@@ -1546,11 +1557,28 @@ static void GetActiveRequests(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   Local<Array> ary = Array::New(args.GetIsolate());
-  int i = 0;
+  Local<Context> ctx = env->context();
+  Local<Function> fn = env->add_properties_by_index_function();
+  Local<Value> argv[5];
+  size_t i = 0;
+
+  for (auto w : *env->req_wrap_queue()) {
+    if (w->persistent().IsEmpty() == false) {
+      argv[i++ % 5] = w->object();
+      if ((i % 5) == 0) {
+        HandleScope scope(env->isolate());
+        fn->Call(ctx, ary, 5, argv).ToLocalChecked();
+        for (size_t j = 0; j < 5; j++) {
+          argv[j] = Local<Value>();
+        }
```

Comment:
Though it still compiles just fine if I do `for (auto&& arg : argv) {`. But TBH I don't know why. Need to go read up on it.